core/lib/entity-classification.js

```diff
@@ -0,0 +1,52 @@
+import {Audit} from '../audits/audit.js';
+import UrlUtils from './url-utils.js';
+import {EntityClassification as ComputedEntityClassification} from '../computed/entity-classification.js';
+
+/**
+ * @param {LH.Artifacts} artifacts
+ * @param {LH.Audit.Context} context
+ */
+async function getEntityClassification(artifacts, context) {
+  const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+  if (!devtoolsLog) return;
+  const classifiedEntities = await ComputedEntityClassification.request(
+    {URL: artifacts.URL, devtoolsLog}, context);
+
+  /** @type {Array<LH.Result.Entity>} */
+  const entities = [];
+  /** @type {Record<string, number>} */
+  const originLUT = {};
```

Comment:
while i've become comfortable with `LUT` after first seeing it in alex's code.. i do think this could be a bit more self-descriptive. 

the integer value for each `origin` key is the index in the entities array, yeah?
so.. we'd call this `originToEntityIndex`

That does feel a bit more obvious to me.

Reply:
`entityIndexByOrigin`?